integration_tests/serverless-forwarder.yml

```diff
@@ -9,15 +9,20 @@ provider:
   region: sa-east-1
   lambdaHashingVersion: 20201221
   logs:
-    restApi: true
-    httpApi: true
-    websocket: true
+    restApi:
+      accessLogging: true
+      executionLogging: false
+    httpApi: false
+    websocket:
```

Comment:
I would change this from `websocket` to `websocketApi` to be consistent with the other options.

Reply:
Oh, this is in Serverless Framework 🤦‍♂️ never mind...
Point still stands, but nothing we can do.